sdk/core/azure-core/src/http/telemetry_policy.cpp

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <http/policy.hpp>
+
+#include <sstream>
```

Comment:
This OSVersion helper should be in utils.hpp and utils.cpp within Azure::Core::Utilities namespace

Reply:
I disagree - I would keep it here until there's another place that needs to use this function.

Reply:
And there's currently no utils.cpp / hpp. That would need to be an internal header, because we don't want to ship it as public API. What would be the other place that uses it? I don't think that we should move this function there at this moment.